addons/base_automation/models/ir_actions_server.py

```diff
@@ -78,6 +78,8 @@ def _compute_name(self):
                         'Create activity: %(activity_name)s',
                         activity_name=action.activity_summary or action.activity_type_id.name
                     )
+                case False:
```

Comment:
avoid that.
The spec of the selection badge should be, in case the field is required.
- The value is set, either because the server sent a default value (as it is the case for ir.actions.server) or because he has selected one option once => then the user cannot unselect the option (the field cannot become unset)
- the value is not set, the field does nothing in particular: no option is set. At worst, the python crashes right away on onchange, this would be the responsibility of the dev to handle this (see the rationale). The expected case is that on save, the field will be marked as missing.

Rationale:
Python side, if the field is required, it should have a default, or the onchange/compute methods should anticipate the case where a field is required but unset. It is not the task of python to handle cases produced by a specific widget client side.